tests/integration/src/test/java/org/apache/pulsar/tests/integration/utils/DockerUtils.java

```diff
@@ -236,6 +239,82 @@ public void onComplete() {
         return result;
     }
 
+    public static ContainerExecResultBytes runCommandWithRawOutput(DockerClient docker,
+            String containerId,
+            String... cmd)
+            throws ContainerExecException {
+        CompletableFuture<Boolean> future = new CompletableFuture<>();
+        String execid = docker.execCreateCmd(containerId)
+                .withCmd(cmd)
+                .withAttachStderr(true)
+                .withAttachStdout(true)
+                .exec()
+                .getId();
+        String cmdString = Arrays.stream(cmd).collect(Collectors.joining(" "));
+        ByteBuf stdout = Unpooled.buffer();
+        ByteBuf stderr = Unpooled.buffer();
+        docker.execStartCmd(execid).withDetach(false)
```

Comment:
There's a lot of code in common with runCommand here. Doesn't need to be in this patch, but it should be factors out to something that takes a ```Consumer<byte[]>```